go/vt/vtctl/vtctl.go

```diff
@@ -2263,7 +2263,7 @@ func commandMoveTables(ctx context.Context, wr *wrangler.Wrangler, subFlags *fla
 
 	autoStart := subFlags.Bool("auto_start", true, "If false, streams will start in the Stopped state and will need to be explicitly started")
 	stopAfterCopy := subFlags.Bool("stop_after_copy", false, "Streams will be stopped once the copy phase is completed")
-	dropConstraints := subFlags.Bool("drop_constraints", false, "If true, tables in the target keyspace will be created without constraints.")
+	dropFKConstraints := subFlags.Bool("drop_fk_constraints", false, "If true, tables in the target keyspace will be created without foreign key constraints.")
```

Comment:
Is it okay to completely replace this flag, and rename it? Are there valid uses of the old flag and path? If not, should that path be cleaned up too?

Reply:
yeah, its not part of any vitess release right now and should be ok to replace, @deepthi or @rohit-nayak-ps to confirm.

Reply:
Since this hasn't crossed a release boundary, it is fine to just rename it.
Though I would suggest calling it `drop_foreign_keys` instead.. @rohit-nayak-ps WDYT?

Reply:
Agree with @deepthi, about calling it `drop_foreign_keys`